_posts/en/newsletters/2023-08-02-newsletter.md

```diff
@@ -0,0 +1,191 @@
+---
+title: 'Bitcoin Optech Newsletter #262'
+permalink: /en/newsletters/2023/08/02/
+name: 2023-08-02-newsletter
+slug: 2023-08-02-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to transcripts of recent LN specification
+meetings and summarizes a thread about the safety of blind MuSig2
+signing.  Also included are our regular sections with descriptions
+of new releases, release candidates, and notable code changes to popular
+Bitcoin infrastructure software.
+
+## News
+
+- **Transcripts of periodic LN specification meetings:** Carla
+  Kirk-Cohen [posted][kc scripts] to the Lightning-Dev mailing list to
+  announce that the last several video conference meetings to discuss
+  changes to the LN specification were transcribed.  The transcripts are
+  now [available][btcscripts spec] on Bitcoin Transcripts.  In related
+  news, as discussed a few weeks ago during the in-person LN developer
+  conference, the '#lightning-dev` IRC chatroom on the [Libera.chat][]
```

Comment:
```suggestion
  conference, the `#lightning-dev` IRC chatroom on the [Libera.chat][]
```

Reply:
Thanks @azuchi , pushed a change to correct